resources/web/wwi/nodes/wbIndexedFaceSet.js

```diff
@@ -0,0 +1,28 @@
+import {WbTriangleMeshGeometry} from './wbTriangleMeshGeometry.js';
+
+class WbIndexedFaceSet extends WbTriangleMeshGeometry {
```

Comment:
```suggestion
export default class WbIndexedFaceSet extends WbTriangleMeshGeometry {
```

Single class files should use `export default`, so you don't need brackets when importing (for example, `import WbGeometry from './wbGeometry.js'`). Also, why the filenames are lowercase, it should be the same as the class names.

Reply:
The old WebotsJS files were following this logical : filenames lowercase, class capitalcase. But I think it is a good idea to change it.

Reply:
changed applied in 37b92a4